plots/measure.js

```diff
@@ -17,11 +17,40 @@
 'use strict';
 
 /* eslint-disable no-console */
-
+const fs = require('fs');
 const path = require('path');
 const parseURL = require('url').parse;
 
 const mkdirp = require('mkdirp');
+const args = require('yargs')
```

Comment:
with the new style of invoking, can you update the readme?

to me, it makes more sense to use the `node script.js` invocation rather than the `npm run` one now that we have a CLI. but no big deal.